server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessorInfoMetrics.java

```diff
@@ -64,11 +65,12 @@ public boolean equals(Object o) {
     if (this == o) return true;
     if (o == null || getClass() != o.getClass()) return false;
     TsFileProcessorInfoMetrics that = (TsFileProcessorInfoMetrics) o;
-    return memCost == that.memCost && Objects.equals(storageGroupName, that.storageGroupName);
+    return Objects.equals(storageGroupName, that.storageGroupName)
```

Comment:
Do we need to override the equal and hashcode of TsFileProcessorInfo?

Reply:
fixed~